mm2src/coins/lightning/ln_utils.rs

```diff
@@ -182,16 +282,21 @@ pub async fn start_lightning(ctx: &MmArc, conf: LightningConf) -> EnableLightnin
 
     let seed: [u8; 32] = ctx.secp256k1_key_pair().private().secret.into();
 
+    // Lock context and wait 1ms before dropping to insure randomness for different coins
+    // when starting multiple lightning nodes for different coins at the same time
+    let background_processor = lightning_ctx.background_processors.lock().await;
     // The current time is used to derive random numbers from the seed where required, to ensure all random generation is unique across restarts.
     let cur = SystemTime::now()
         .duration_since(SystemTime::UNIX_EPOCH)
         .map_to_mm(|e| EnableLightningError::SystemTimeError(e.to_string()))?;
 
     // Initialize the KeysManager
     let keys_manager = Arc::new(KeysManager::new(&seed, cur.as_secs(), cur.subsec_nanos()));
+    Timer::sleep_ms(1).await;
```

Comment:
This is related to this comment in the previous PR https://github.com/KomodoPlatform/atomicDEX-API/pull/1103#discussion_r729851659, I guess there will be no `background_processors` mutex anyways when implementing the `LightningCoin` struct suggested in the first comment so this will be removed.